pkg/controller/queuejobresources/genericresource/genericresource.go

```diff
@@ -426,8 +438,11 @@ func getPodResources(pod arbv1.CustomPodResourceTemplate) (resource *clusterstat
 	return req
 }
 
+// to get allocated resources in a container and its replicas
 func getContainerResources(container v1.Container, replicas float64) *clusterstateapi.Resource {
+	// track the requested resource in a container
```

Comment:
1. Suggest capitalize comment sentences.
2. A semantic suggested change to help clarify, `Track the requested resource of a container`.  Resources are not really inside a container but instead configuration/parameter/field of a container definition.
3. Better yet maybe: `Make a copy of the requested resources of a container`.   Hopefully this help describe what is intended for the code below.  :D